### PR TITLE
Update tutorials to mention AMP

### PR DESCRIPTION
**Pull request by tbouron**

### CODE REVIEW

File: tutorials/swarm-cluster.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ title: Swarm Cluster Tutorial
 ---
 
 ### Introduction
-This tutorial is focused on deploying a production ready swarm cluster.
+This tutorial is focused on deploying a production ready Docker Swarm.
 
 ### Pre-requisites
-This tutorial assumes you have [installed](https://brooklyn.apache.org/v/latest/start/running.html) Apache Brooklyn, and the [Apache Brooklyn CLI](https://brooklyn.apache.org/v/latest/ops/cli/index.html).
+This tutorial assumes you have installed [Apacke Brooklyn](https://brooklyn.apache.org/v/latest/start/running.html) or [Cloudsoft AMP](http://docs.cloudsoft.io/ccs/tutorial/tutorial-get-amp-running.html), as well as the [Apache Brooklyn CLI](https://brooklyn.apache.org/v/latest/ops/cli/index.html).
```
Comment (drigodwin) [inline tutorials/swarm-cluster.md]: Apacke -> Apache

File: tutorials/kubernetes-cluster.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ This tutorial is focused on deploying a production ready Kubernetes cluster.
 The production ready Kubernetes cluster is comprised of the following components:
 
 ### Pre-requisites
-This tutorial assumes you have [installed](https://brooklyn.apache.org/v/latest/start/running.html) Apache Brooklyn, and the [Apache Brooklyn CLI](https://brooklyn.apache.org/v/latest/ops/cli/index.html).
+This tutorial assumes you have installed [Apacke Brooklyn](https://brooklyn.apache.org/v/latest/start/running.html) or [Cloudsoft AMP](http://docs.cloudsoft.io/ccs/tutorial/tutorial-get-amp-running.html), as well as the [Apache Brooklyn CLI](https://brooklyn.apache.org/v/latest/ops/cli/index.html).
```
Comment (drigodwin) [inline tutorials/kubernetes-cluster.md]: Apacke -> Apache

**Comment by drigodwin** [comment]
Looks good aside from the typo

**Comment by tbouron** [comment]
@drigodwin Comments addressed

**Pull request merged**